e2e_testing/torchscript/basic.py

```diff
@@ -371,10 +371,32 @@ def EmbeddingModule_basic(module, tu: TestUtils):
     module.forward(torch.randint(100, (3, 3)))
 
 
+class SoftmaxBackwardModule(torch.nn.Module):
```

Comment:
Instead of writing it in `basic.py` we can have another file named `backprop_ops.py` where we can write tests for backward ops. It's easy to navigate.

Reply:
sounds good!

Reply:
sounds good!